index.js

```diff
@@ -140,6 +140,14 @@ function parseStyles(result, styles, options, state, media) {
             return
           }
 
+          if (options.filter) {
+            let filtered = options.filter(stmt.uri);
+
+            if (filtered === false) {
+              return;
+            }
```

Comment:
I do a strict `false` check so blocking is deliberate. When the filter has no return-statement and the return value is `undefined`, the filter will have no effect.

I can shorten it to `if (options.filter(stmt.uri) === false)` if you want but I typically opt for the verbose option as readability -> maintainability especially in collaborative projects.